test/helpers.ts

```diff
@@ -59,3 +59,7 @@ export function getGanacheControls(rpc: JsonRpc): GanacheControls {
 		},
 	}
 }
+
+export async function mineBlock(actor: Actor) {
+	return actor.rpc.sendEth(actor.signer.address, 0n)
+}
```

Comment:
Hax.  I wonder if we should name this `triggerInstasealBlock` to help distinguish it from the other `mine` function in here.

Reply:
much better name, since we aren't mining at all